charon/config.py

```diff
@@ -39,6 +40,15 @@ def __init__(self, data: Dict):
         self.__ignore_signature_suffix: Dict = data.get("ignore_signature_suffix", None)
         self.__signature_command: str = data.get("detach_signature_command", None)
         self.__aws_cf_enable: bool = data.get("aws_cf_enable", False)
+        self.__amqp_queue: str = data.get("amqp_queue", None)
+        self.__sign_result_loc: str = data.get("sign_result_loc", None)
+        self.__quay_radas_auth_enabled: bool = data.get("quay_radas_auth_enabled", False)
+        self.__quay_radas_registry: str = data.get("quay_radas_registry", None)
+        self.__quay_radas_username: str = data.get("quay_radas_username", None)
+        self.__quay_radas_password: str = data.get("quay_radas_password", None)
```

Comment:
These quay specified items should not be set through configuration. they should be specified through cmd flag.

Reply:
@ligangty, if through cmd flags, might need to transfer the flags values when setup umb client(sign request send) with umb listener register, since this will be used for on_message. But could you help to explain why it needs to be specified through cmd flag?

Reply:
This registry is a variable one but not fixed for different packages. For example, for different qurakus versions, the registry will have different version tags in quay.

Reply:
and for the username and password, maybe we will use another way like  registry-config file. See "oras --registry-config" for details.

Reply:
@ligangty Just for sure, you mean the registry host(https://quay.io/repository/signing/radas) will be changed for different products? This properties are just used for login, it means oras just needs hostname. For tags, it should be used for pull and we could get that from the result_reference url.
So,
1. Maybe we don't need quay_radas_registry, just get the hostname from the result_reference url extracting, CMIMW.
2. For the registry-config file, does it store in any env loc for different products, or still from user cmd input flags?

Reply:
> Maybe we don't need quay_radas_registry, just get the hostname from the result_reference url extracting, CMIMW.
Agree with this.

> For the registry-config file, does it store in any env loc for different products, or still from user cmd input flags?
FWIK now, the registry is a public one which don't need auth. So maybe for first version we will not care auth issue. For the registry-config file, I think it will be a configmap or secret later in openshift managed workspace.

Reply:
So, I just file a quay_radas_registry_config.py and we could enhance more if it needs in future, WDTY?

Reply:
Maybe you can directly add them to the RadasConfig class in my new merged PR?

Reply:
@ligangty Sure, will adjust that.